controllers/articles.controller.js

```diff
@@ -17,3 +22,19 @@ exports.getAllArticles = (req, res, next) => {
         })
         .catch(next);
 };
+
+exports.getCommentsByArticleId = (req, res, next) => {
+    let { article_id } = req.params;
+
+    const promises = [selectCommentsByArticleId(article_id)];
+
+    if (article_id) {
+        promises.push(checkArticleExist(article_id));
+    }
+
+    return Promise.all(promises)
+        .then((comments) => {
+            res.status(200).send({ comments: comments[0] });
```

Comment:
The comments[0] should really only happen in the model 😄

Reply:
About the last comment you gave me, it is a Promise.all() that's why I did that and how could I make it happen in side model?? Ty.

Reply:
> The comments[0] should really only happen in the model 😄

About the last comment you gave me, it is a Promise.all() that's why I did that and how could I make it happen in side model?? Ty.